installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -0,0 +1,58 @@
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[Installing a cluster on vSphere]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with no customization.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc#installing-vsphere-installer-provisioned-customizations[Installing a cluster on vSphere with customizations]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with the default customization options.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installing-vsphere-installer-provisioned-network-customizations[Installing a cluster on vSphere with network customizations]**: You can install {product-title} on installer-provisioned vSphere infrastructure, with network customizations. You can customize your OpenShift Container Platform network configuration during installation, so that your cluster can coexist with your existing IP address allocations and adhere to your network requirements.
+
+* **xref:../../installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc#installing-restricted-networks-installer-provisioned-vsphere[Installing a cluster on vSphere in a restricted network]**: You can install a cluster on VMware vSphere infrastructure in a restricted network by creating an internal mirror of the installation release content.
+ You can use this method to deploy {product-title} on an internal network that is not visible to the Internet.
+
+=== User-provisioned infrastructure installation of {product-title} on vSphere
+
+User-provisioned infrastructure requires the user to provision all resources required by {product-title}.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere.adoc#[Installing a cluster on vSphere with user-provisioned infrastructure]**: You can install {product-title} on VMware vSphere infrastructure that you provision.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-network-customizations.adoc#installing-vsphere-network-customizations[Installing a cluster on vSphere with network customizations with user-provisioned infrastructure]**: You can install {product-title} on VMware vSphere infrastructure that you provision with customized network configuration options.
+
+* **xref:../../installing/installing_vsphere/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[Installing a cluster on vSphere in a restricted network with user-provisioned infrastructure]**: {product-title} can be installed on VMware vSphere infrastructure that you provision in a restricted network.
```

Comment:
s/{product-title} can be installed/you can install {product-title} for consistency between the bullets.

Reply:
Do you want to add _by creating an internal mirror of the installation release content.
 You can use this method to deploy {product-title} on an internal network that is not visible to the Internet._ to this bullet also, to define what a restricted network is?

Reply:
First item: Done, thank you!
Second item: We didn't want to equate "restricted network" with "not visible to the internet", because that gets into a gray area with the support engineers. Support does not want to be on the hook for internal mirrored content (not quay) so we stayed away from that terminology.

Reply:
In that case, did you want to [remove the other reference to _not visible to the internet_](https://github.com/openshift/openshift-docs/pull/33795/files#diff-26f75f0dd9884af7fd0cf84cafa1848f520053e5b7cc5c53fe27087408992060R44)?